lib/pleaserun/platform/base.rb

```diff
@@ -210,4 +232,18 @@ def log_path_stdout
     filename = log_file_stdout unless log_file_stdout.nil?
     File.join(log_directory.chomp("/"), filename)
   end
+
+  def parsed_environment_variables
+    return {} if environment_file.nil?
+    return {} unless File.exist?(environment_file)
+    Dotenv::Parser.call(File.open(environment_file, "rb:bom|utf-8", &:read))
+  end
+
+  def all_environment_variables
+    variables = parsed_environment_variables.merge(environment_variables)
```

Comment:
Let me just do gentle checking to ensure the code doesn't do something silly here.